app/config.go

```diff
@@ -9,17 +9,18 @@ import (
 	"github.com/TIBCOSoftware/flogo-lib/core/action"
 	"github.com/TIBCOSoftware/flogo-lib/core/data"
 	"github.com/TIBCOSoftware/flogo-lib/core/trigger"
+	"github.com/pkg/errors"
 )
 
 // App is the configuration for the App
 type Config struct {
-	Name        string                 `json:"name"`
-	Type        string                 `json:"type"`
-	Version     string                 `json:"version"`
-	Description string                 `json:"description"`
-	Properties  map[string]interface{} `json:"properties"`
-	Triggers    []*trigger.Config      `json:"triggers"`
-	Resources   []*resource.Config     `json:"resources"`
+	Name        string             `json:"name"`
+	Type        string             `json:"type"`
+	Version     string             `json:"version"`
+	Description string             `json:"description"`
+	Properties  interface{}        `json:"properties"`
```

Comment:
not sure if anyone actually used the previous property support, wasn't really advertised.  Maybe we should just make it []Attribute, @mellistibco thoughts?

Reply:
I think its ok to just discard the previous property support, I suspect very few people were using it...

Reply:
@mellistibco @fm-tibco : This is fixed in now. Please review updated PR.